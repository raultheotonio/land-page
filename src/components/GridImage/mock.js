export default {
  background: false,
  title: 'My Grid Image',
  description: 'Description',
  grid: [
    {
      altText: 'alt text 1',
      srcImg: 'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      altText: 'alt text 2',
      srcImg: 'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      altText: 'alt text 3',
      srcImg: 'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      altText: 'alt text 4',
      srcImg: 'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      altText: 'alt text 5',
      srcImg: 'https://source.unsplash.com/random/800x800?r=5',
    },
  ],
};
