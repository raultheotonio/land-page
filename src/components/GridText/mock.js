export default {
  title: 'title',
  description: 'description',
  background: false,
  grid: [
    {
      title: 'grid 1',
      description: 'grid description',
    },
    {
      title: 'grid 2',
      description: 'grid description',
    },
    {
      title: 'grid 3',
      description: 'grid description',
    },
  ],
};
