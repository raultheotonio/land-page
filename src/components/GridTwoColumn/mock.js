export default {
  title: 'Grid two columns',
  text: 'Somente um teste aqui por favor',
  background: true,
  srcImg: 'assets/images/javascript.svg',
};
