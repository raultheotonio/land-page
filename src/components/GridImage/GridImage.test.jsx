import { GridImage } from '.';
import renderTheme from '../../utils/RenderTheme';
import mock from './mock';

describe('<GridImage />', () => {
  it('should render', () => {
    const { container } = renderTheme(<GridImage {...mock} />);

    expect(container).toMatchSnapshot();
  });
});
