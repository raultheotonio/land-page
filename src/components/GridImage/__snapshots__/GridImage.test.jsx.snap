// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridImage /> should render 1`] = `
<div>
  <div
    class="sc-gswNZR bzkTFK"
  >
    <div
      class="sc-dkrFOg AkWbM"
    >
      <div
        class="sc-jSUZER iBJmju"
      >
        <div>
          <h2
            class="sc-bcXHqe icqeWr"
          >
            My Grid Image
          </h2>
        </div>
        <p
          class="sc-eDvSVe gGWFcm"
        >
          Description
        </p>
        <div
          class="sc-gKPRtg keBiKO"
        >
          <div
            class="sc-iBYQkv fyqPic"
          >
            <img
              alt="alt text 1"
              class="sc-ftTHYK kHftfy"
              src="https://source.unsplash.com/random/800x800?r=1"
            />
          </div>
          <div
            class="sc-iBYQkv fyqPic"
          >
            <img
              alt="alt text 2"
              class="sc-ftTHYK kHftfy"
              src="https://source.unsplash.com/random/800x800?r=2"
            />
          </div>
          <div
            class="sc-iBYQkv fyqPic"
          >
            <img
              alt="alt text 3"
              class="sc-ftTHYK kHftfy"
              src="https://source.unsplash.com/random/800x800?r=3"
            />
          </div>
          <div
            class="sc-iBYQkv fyqPic"
          >
            <img
              alt="alt text 4"
              class="sc-ftTHYK kHftfy"
              src="https://source.unsplash.com/random/800x800?r=4"
            />
          </div>
          <div
            class="sc-iBYQkv fyqPic"
          >
            <img
              alt="alt text 5"
              class="sc-ftTHYK kHftfy"
              src="https://source.unsplash.com/random/800x800?r=5"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
